examples/shader/post_processing.rs

```diff
@@ -162,20 +78,254 @@ fn main_camera_cube_rotator_system(
+                RenderTarget::Window(window_id) => {
+                    let window = windows.get(*window_id).expect("ColorBlindnessCamera is rendering to a window, but this window could not be found");
+                    option_window_id = Some(*window_id);
+                    Extent3d {
+                        width: window.physical_width(),
+                        height: window.physical_height(),
+                        ..Default::default()
+                    }
+                }
+                RenderTarget::Image(handle) => {
+                    let image = images.get(handle).expect(
+                    "ColorBlindnessCamera is rendering to an Image, but this Image could not be found",
+                );
+                    image.texture_descriptor.size
+                }
+            };
+
+            // This is the texture that will be rendered to.
+            let mut image = Image {
+                texture_descriptor: TextureDescriptor {
+                    label: None,
+                    size,
+                    dimension: TextureDimension::D2,
+                    format: TextureFormat::bevy_default(),
+                    mip_level_count: 1,
+                    sample_count: 1,
+                    usage: TextureUsages::TEXTURE_BINDING
+                        | TextureUsages::COPY_DST
+                        | TextureUsages::RENDER_ATTACHMENT,
+                },
+                ..Default::default()
+            };
+
+            // fill image.data with zeroes
+            image.resize(size);
+
+            let image_handle = images.add(image);
+
+            // This specifies the layer used for the post processing camera, which will be attached to the post processing camera and 2d fullscreen triangle.
+            let post_processing_pass_layer =
+                RenderLayers::layer((RenderLayers::TOTAL_LAYERS - 1) as u8);
+            let half_extents = Vec2::new(size.width as f32 / 2f32, size.height as f32 / 2f32);
+            let mut triangle_mesh = Mesh::new(PrimitiveTopology::TriangleList);
+            // NOTE: positions are actually not used because the vertex shader maps UV and clip space.
+            triangle_mesh.insert_attribute(
+                Mesh::ATTRIBUTE_POSITION,
+                vec![
+                    [-half_extents.x, -half_extents.y, 0.0],
+                    [half_extents.x * 3f32, -half_extents.y, 0.0],
+                    [-half_extents.x, half_extents.y * 3f32, 0.0],
+                ],
+            );
+            triangle_mesh.set_indices(Some(Indices::U32(vec![0, 1, 2])));
+            triangle_mesh.insert_attribute(
+                Mesh::ATTRIBUTE_NORMAL,
+                vec![[0.0, 0.0, 1.0], [0.0, 0.0, 1.0], [0.0, 0.0, 1.0]],
+            );
+
+            triangle_mesh.insert_attribute(
+                Mesh::ATTRIBUTE_UV_0,
+                vec![[2.0, 0.0], [0.0, 2.0], [0.0, 0.0]],
+            );
+            let triangle_handle = meshes.add(triangle_mesh);
```

Comment:
I've written that positions are not used, but it's probably a lie, as if I change them a bit I notice it's not working properly ; I'm open to suggestions to explain what's happening here.

Reply:
At least on my end (macos/m1), things seem to work properly as long as some positions are set, even
```rust
triangle_mesh.insert_attribute(
    Mesh::ATTRIBUTE_POSITION,
    vec![[0., 0., 0.], [0., 0., 0.], [0., 0., 0.]],
);
```